src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -321,6 +328,206 @@ public async Task<GetTransactionModel> GetTransactionAsync(string txid)
             return model;
         }
 
+        [ActionName("listaddressgroupings")]
+        [ActionDescription("Returns a list of grouped addresses which have had their common ownership made public by common use as inputs or as the resulting change in past transactions.")]
+        public async Task<AddressGroupingModel[]> ListAddressGroupings(string walletName, CoinType coinType)
```

Comment:
The wallet manager has the `coinType` in it, is it necessary to pass this?

Reply:
Looks like WalletManager.GetWallet() requires a valid wallet name. We will want this to be compatible with bitcoin signature which takes in no parameters https://bitcoincore.org/en/doc/0.17.0/rpc/wallet/listaddressgroupings/ so should probably do something similar to the other RPC methods in WalletRPCController. I think most of the call ``` this.GetAccount(); ``` to get a reference to the wallet.

Reply:
@zeptin Instead making changes to the `WalletManager` I will check `Network`s CoinType.

@mikedennis thanks for the heads up, I will ensure that the signature matches.